src/main/java/com/github/jikoo/enchantableblocks/util/Cache.java

```diff
@@ -88,7 +88,7 @@ private Cache(final long retention, final BiFunction<K, Boolean, V> load,
 	 */
 	public void put(final K key, final V value) {
 		// Run lazy check to clean cache
-		this.lazyCheck();
+		//this.lazyCheck();
```

Comment:
If you're going to comment out a line of code, it should just be deleted - that's what version control is for. Several cases of this.